src/org/opendatakit/briefcase/ui/reused/source/RemoteServerDialog.java

```diff
@@ -68,9 +68,9 @@ private void showError(String error, String title) {
     );
   }
 
-  static RemoteServerDialog empty(RemoteServer.Test serverTester) {
+  static RemoteServerDialog empty(RemoteServer.Test serverTester, String requiredPermission) {
```

Comment:
I think these should use an enum and not Strings, but I don’t want that to hold up the beta.
`String requiredPermission`

What’s that derogatory joke? _string_ typing
http://wiki.c2.com/?StringlyTyped